net/tcpflow/Portfile

```diff
@@ -24,18 +26,17 @@ long_description    tcpflow is a program that captures data transmitted as \
                     streams regardless of retransmissions or out-of-order \
                     delivery.
 
-depends_build       port:autoconf \
-                    port:automake \
-                    port:boost
```

Comment:
@barracuda156 are you sure that it doesn't need boost anymore?

Reply:
@catap I added `boost 1.0` PG. Doesn’t it replace explicit dependency?

Reply:
@barracuda156 indeed, I've missed that!

Reply:
but if it is only a build dependency you should specify that using `boost.depends_type` as by default it is set as a library dependency

Reply:
@reneeotten Thank you!